docker-compose.yml

```diff
@@ -1,199 +1,174 @@
 version: "3.8"
 
 networks:
-    omotes:
+  omotes:
+    driver: bridge
+    name: omotes
 
 volumes:
   db-data:
   broker-data:
   influxdb_storage:
 
 services:
-    rabbitmq:
```

Comment:
What is changed on these lines? The newline char? Or did you use other indenting?

Reply:
The indenting changed due to autoreformat.
Previously the indent was not inline with the `volumes:` item above.